src/libraries/System.Private.CoreLib/src/Interop/Windows/Kernel32/Interop.SECURITY_ATTRIBUTES.cs

```diff
@@ -16,5 +16,24 @@ internal struct SECURITY_ATTRIBUTES
             internal IntPtr lpSecurityDescriptor;
             internal BOOL bInheritHandle;
         }
+
+        internal static unsafe SECURITY_ATTRIBUTES GetSecAttrs(BOOL inheritHandle = default)
+        {
+            return new SECURITY_ATTRIBUTES
+            {
+                nLength = (uint)sizeof(SECURITY_ATTRIBUTES),
+                bInheritHandle = inheritHandle
+            };
+        }
+
+        internal static unsafe SECURITY_ATTRIBUTES GetSecAttrs(byte* pSecurityDescriptor, BOOL inheritHandle = default)
```

Comment:
If you are making a helper method out of this, the argument should be `bool` so that callers do not have to normalize it.